pyesbulk/__init__.py

```diff
@@ -285,8 +288,7 @@ def actions_tracking_closure(cl_actions):
             status = resp["status"]
         except KeyError as e:
             assert not ok, f"ok = {ok!r}, e = {e!r}"
-            # Limit the length of the error message.
-            logger.error("{!r}"[:_MAX_ERRMSG_LENGTH], e)
+            logger.error("{!r}", e)
```

Comment:
So back to no truncation at all? How big can a KeyError payload get, though?

Reply:
I am not sure why I added it in the first place.  It can't get that big.